source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/index.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { match as Match } from 'react-router';
+import { compose } from 'recompose';
+import { withData as withOrbit } from 'react-orbitjs';
+
+import { OrganizationAttributes } from '@data/models/organization';
+import {
+  OrganizationResource,
+  OrganizationStoreResource,
+  StoreResource
+} from '@data';
+
+import StoreMultiSelect from '@ui/components/inputs/store-multi-select';
+import { withTranslations, i18nProps } from '@lib/i18n';
+
+export const pathName = '/organizations/:orgId/settings/stores';
+
+export interface Params {
+  orgId: string;
+}
+
+export interface IProps {
+  match: Match<Params>;
+  update: (payload: OrganizationAttributes) => void;
+  updateOrganizationStore: (payload: StoreResource) => void;
+  organization: OrganizationResource;
+  organizationStores: OrganizationStoreResource[];
+}
+
+type IOwnProps =
+  & IProps
+  & i18nProps;
+
+const mapRecordsToProps = (passedProps) => {
+  const { organization } = passedProps;
+  const { type, id } = organization;
+
+  return {
+    organizationStores: q => q.findRelatedRecords({ type, id }, 'organizationStores')
```

Comment:
any reason to not just pass the organization here?

Reply:
changed